### PR TITLE
Feature/editable data points

### PR DESCRIPTION
**Pull request by j-or**

### CODE REVIEW

File: reducers/reducers.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export const CATEGORICAL_VARIABLE_DELETED = 'CATEGORICAL_VARIABLE_DELETED'
 export const CONFIGURATION_UPDATED = 'CONFIGURATION_UPDATED'
 export const RESULT_REGISTERED = 'RESULT_REGISTERED'
 export const DATA_POINTS_ADDED = 'DATA_POINTS_ADDED'
+export const DATA_POINTS_UPDATED = 'DATA_POINTS_EDITED'
```
Comment (langdal) [inline reducers/reducers.ts:14]: Shouldn't the value match the variable name for DATA_POINTS_UPDATED?

**Review by langdal — CHANGES_REQUESTED** [review]
Looks good overall.

One issue that should probably be fixed before merging is when clicking on the "X" in the new data point line it dissapears and there are no way to get the line back.

File: reducers/reducers.ts (hunk 1 above)
Comment (j-or) [reply reducers/reducers.ts]: Well spotted. Fixed.

**Comment by j-or** [comment]
> Looks good overall.
> 
> One issue that should probably be fixed before merging is when clicking on the "X" in the new data point line it dissapears and there are no way to get the line back.

Agreed, thanks. Disappearing row now fixed and test added that covers the scenario.

**Review by langdal — APPROVED** [review]

**Pull request merged**